CHANGELOG.md

```diff
@@ -8,7 +8,11 @@ All notable changes to this project will be documented in this file.
 
 - Various documentation of the CRD ([#510]).
 
-[#510]: https://github.com/stackabletech/trino-operator/pull/510
+### Fixed
+
+- Add `core-site.xml` when configuring HDFS connection ([#526]).
+
+[#526]: https://github.com/stackabletech/trino-operator/pull/526
```

Comment:
You removed the 510 reference by accident?
```suggestion
### Fixed

- Add `core-site.xml` when configuring HDFS connection ([#526]).

[#510]: https://github.com/stackabletech/trino-operator/pull/510
[#526]: https://github.com/stackabletech/trino-operator/pull/526
```

Reply:
Yes, thanks!